cli/src/lib.rs

```diff
@@ -137,22 +146,18 @@ where
     /// - Reading telemetry configs
     /// - telemetry setup
     /// - Initialization of [`Sumeragi`]
-    #[allow(clippy::non_ascii_literal)]
     pub async fn with_genesis(
         genesis: Option<G>,
         config: Configuration,
         instruction_validator: IsInstructionAllowedBoxed<K::World>,
         query_validator: IsQueryAllowedBoxed<K::World>,
         broker: Broker,
+        telemetry: Option<iroha_logger::Telemetries>,
     ) -> Result<Self> {
         if !config.disable_panic_terminal_colors {
-            if let Err(e) = color_eyre::install() {
-                iroha_logger::error!("Tried to install eyre_hook twice: {:?}", e);
-            }
+            // Ignoring error if `iroha_logger` has already installed colors
+            let _color_res = color_eyre::install();
```

Comment:
I think we should still log it. Maybe `debug`.

Reply:
When I was running Iroha I saw this error in the log. And after some new commits and rebasing the error dissapeared... So I think, I'll bring back `error` log here